.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.7
```

Comment:
Is it still necessary as a standalone file if it's specified in the `Gemfile`? CircleCI says:

> We infer your Ruby version from your .rvmrc file, .ruby-version file, or Gemfile

I don't know what happens with dynamic definition.